docs/source/release_notes/release-notes.rst

```diff
@@ -98,132 +106,157 @@ develop
            Now, we only request the latest version of the specific column requested, if only one column is requested. Requesting multiple columns still results in the previous behavior, however this will also be optimized in a future release.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2480>`__)
 
-    *    - |fixed|
-         - ``SerializableErrorDecoder`` will decode error properly instead of throwing ``NullPointerException``.
-
     *    - |deprecated| |improved|
          - ``SerializableTransactionManager`` is now created via an immutable builder instead of a long list of individual arguments. Use ``TransactionManagers.builder()``
            to get the builder and once completely configured, build the transaction manager via the builder's ``.buildSerializable()`` method.
            The existing ``create`` methods are deprecated and will be removed by November 15th, 2017.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2459>`__)
 
-    *    - |fixed|
-         - Async Initialization now works with TimeLock Server.
-           Previously, for Cassandra we would attempt to immediately migrate the timestamp bound from Cassandra to TimeLock on startup, which would fail if either of them was unavailable.
-           For DBKVS or other key-value services, we would attempt to ping TimeLock on startup, which would fail if TimeLock was unavailable (though the KVS need not be available).
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2518>`__)
+    *    - |devbreak|
+         - ``TransactionManagers.builder()`` no longer has a ``callingClass(..)`` method and now requires the consumer to directly specify their user agent via the previously optional method ``userAgent(..)``. All of the ``TransactionManagers.create(..)`` methods are still deprecated.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2542>`__)
 
-    *    - |fixed|
-         - ``AsyncInitializer`` now shuts down its executor after initialization has completed.
-           Previously, the executor service wasn't shut down, which could lead to the initializer thread hanging around unnecessarily.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2518>`__)
+    *    - |improved|
+         - The duration between attempts of whitelist Cassandra nodes was reduced from 5 minutes to 2 minutes, and the minimum period a node is blacklisted for was reduced from 2 minutes to 30 seconds.
+           This means we check the health of a blacklisted Cassandra node and whitelist it faster than before.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2543>`__)
 
-    *    - |fixed|
+    *    - |improved| |devbreak|
+         - Size of the transaction cache is now configurable. It is not anticipated end users will need to touch this;
```

Comment:
nit: `The size...`

Reply:
You should note the new parameter name and default value (see the below note), and also that while it's in the `runtime` block, it actually does not yet live reload (link to the relevant issue?)